cvent-technology-radar.csv

```diff
@@ -159,3 +159,5 @@ Axe,Adopt,Tools,FALSE,"<p>Axe is a tool integrated to both Blucumber and WDIO. A
 Blucumber,Hold,Tools,FALSE,<p>Blucumber is an internal Cvent tool used for building/running selenium based test automation on our legacy applications. For newer applications we're favoring webdriver.io</p>
 JAWS Screen Reader,Adopt,Tools,FALSE,<p>We leverage JAWS as a screen reader for testing accessibility.</p>
 Prisma,Hold,languages-and-frameworks,TRUE,<p>Prisma is an interesting ORM that works well with graphql + typescript but has several short comings when it comes to how it handles transactions and joins. Prisma also uses a sidecar process to execute queries that severely hinders observability tools like Datadog. The side car process complicates connection pooling and IAM authentication and the sql generated is un friendly to RDS Proxy making it an unsuitable tool for serverless deployments. The approach we've taken to handle IAM results in memory leaks. For this reason we’re putting this technology on Hold and looking for alternatives.</p>
+Nx,Adopt,Tools,TRUE,<p>Nx is a build system designed to operate monorepos at scale. It adds computational caching and a host of other useful features to prevent the need to run every package in the monorepo every time.</p>
+Dagger,Assess,Tools,TRUE,<p>Dagger is a devkit built on top of Docker's BuildKit for building portable CI/CD pipelines that can be run on any CI system or locally</p>
```

Comment:
Add a trailing newline 😄

Reply:
Psssh, this is how it exported the CSV for me. 😛